playwright/pages/dialogs/addNavigationMenuItemDialog.ts

```diff
@@ -14,20 +14,37 @@ export class AddNavigationMenuItemDialog extends BasePage {
     super(page);
   }
 
-  async selectLinkOption(option: string, optionName: string) {
+  async selectLinkTypeOption(linkType: string) {
     await this.menuLinkType.click();
     await this.waitForDOMToFullyLoad();
-    await this.menuLinkOptions.filter({ hasText: "Categories" }).waitFor({ state: "visible" });
-    await this.menuLinkOptions.filter({ hasText: "Collections" }).waitFor({ state: "visible" });
-    await this.menuLinkOptions.filter({ hasText: "Pages" }).waitFor({ state: "visible" });
-    await expect(this.menuLinkOptions.filter({ hasText: option })).toBeEnabled();
-    await this.menuLinkOptions.filter({ hasText: option }).click({ force: true });
-    await this.waitForDOMToFullyLoad();
+
+    // Ensure the link type option is visible and select it
```

Comment:
LLM models generate usually tons on comments, explaining what they do - it's useful when composing the final solution, but usually redundant when used in the final codebase.